.github/workflows/docker.yml

```diff
@@ -38,9 +38,13 @@ jobs:
           git diff
           echo "===== END ====="
 
-      - name: Build docker image
+      - name: Build test docker image
         run: |
-          docker build -t gluwa/creditcoin3-staking-dashboard .
+          docker build --build-arg="TARGET_NETWORK=test" -t gluwa/creditcoin3-staking-dashboard .
+
+      - name: Build prod docker image
+        run: |
+          docker build --build-arg="TARGET_NETWORK=prod" -t gluwa/creditcoin3-staking-dashboard .
```

Comment:
Can you swap this line with the line before it.  It is breaking the PR preview because it removes the "CreditcoinLocal" network definition which is used for testing.